Core/Configuration/PluginConfig.cs

```diff
@@ -290,9 +290,9 @@ public void ForceLoadPlugins()
         /// </summary>
         public ILogManager LogManager { get { return _logManager; } set { _logManager = value; if (LoggingAvailable != null && value != null) LoggingAvailable(value); } }
 
-        public delegate void LoggingAvaialableEvent(ILogManager logger);
+        public delegate void LoggingAvailableEvent(ILogManager logger);
```

Comment:
This is potentially an API change

Reply:
I don't think we can make this change, it's likely in use

Reply:
Not OK to change.